MREUnityRuntime/MREUnityRuntimeLib/App/IMixedRealityExtensionApp.cs

```diff
@@ -83,7 +84,7 @@ public interface IMixedRealityExtensionApp
 		/// <summary>
 		/// The url of the MRE server. Only valid after `Startup` has been called.
```

Comment:
>url [](start = 10, length = 3)

Uri or Url?  The name of the property has now changed to Uri, so would be good if this comment aligned to the name and whether it is a url or uri.

Reply:
Good call, I'll update this.